content-review/impl/vericite/src/main/java/org/sakaiproject/contentreview/vericite/ContentReviewServiceVeriCite.java

```diff
@@ -910,6 +911,20 @@ private void uploadExternalContent(String urlString, byte[] data){
 			connection=(HttpURLConnection) url.openConnection();
 			connection.setDoOutput(true);
 			connection.setRequestMethod("PUT");
+
+			String headerString = ObjectUtils.toString(headers);
```

Comment:
Thanks, didn't notice the deprecation warning in IDE. I think ObjectUtils came from commons-lang, which Sakai uses 2.6 (where it wasn't deprecated), as opposed to importing from commons-lang3, which is 3.5 where it is deprecated. I've replaced this with Objects.toString(headers, ""); I need to be ensure the value isn't null here.